### PR TITLE
0.2.4 stable

### PR DESCRIPTION
**Pull request by diptib10**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 source 'https://rubygems.org'
 
-gem 'spree', github: 'spree/spree', branch: '2-4-stable'
-gem 'spree_gateway', github: 'spree/spree_gateway', branch: '2-4-stable'
-gem 'spree_auth_devise', github: 'spree/spree_auth_devise', branch: '2-4-stable'
+gem 'spree', github: 'spree/spree', branch: 'master'
+gem 'spree_gateway', github: 'spree/spree_gateway', branch: 'master'
+gem 'spree_auth_devise', github: 'spree/spree_auth_devise', branch: 'master'
```
Comment (curtiscook) [inline Gemfile]: Use :tag & tag the release when done

File: app/views/spree/checkout/payment/_affirm.html.erb [hunk 2]
```diff
@@ -58,10 +58,14 @@
       },
 
 
+      metadata: {
+            "platform_type": "spree_affirm",
```
Comment (curtiscook) [inline app/views/spree/checkout/payment/_affirm.html.erb]: "Spree Commerce"

File: spree_affirm.gemspec [hunk 3]
```diff
@@ -7,12 +7,12 @@ Gem::Specification.new do |s|
...
-  s.description = 'Affirm payment Gateway for spree'
+  s.description = 'Affirm payment Gateway for Spree'
   s.required_ruby_version = '>= 1.9.3'
 
-  s.author    = 'Yu Pan'
-  s.email     = 'yu.pan@affirm.com'
-  s.homepage  = 'http://www.spreecommerce.com'
+  s.author    = 'Affirm'
+  s.email     = 'mts@affirm.com'
+  s.homepage  = 'http://www.affirm.com'
```
Comment (curtiscook) [inline spree_affirm.gemspec]: https://www.affirm.com/business

File: Gemfile (hunk 1 above)
Comment (curtiscook) [reply Gemfile]: v0.2.24

File: app/models/spree/gateway/affirm.rb [hunk 4]
```diff
@@ -2,8 +2,8 @@ module Spree
   class Gateway::Affirm < Gateway
     preference :api_key, :string
     preference :secret_key, :string
-    preference :server, :string, default: 'www.affirm.com'
-    preference :product_key, :string
+    preference :test_mode, :boolean, default: true
+    preference :server, :string, default: 'sandbox.affirm.com'
```
Comment (curtiscook) [inline app/models/spree/gateway/affirm.rb]: This is API base URL (for clarity)

File: app/models/spree/gateway/affirm.rb [hunk 5]
```diff
@@ -2,8 +2,8 @@ module Spree
   class Gateway::Affirm < Gateway
     preference :api_key, :string
     preference :secret_key, :string
-    preference :server, :string, default: 'www.affirm.com'
-    preference :product_key, :string
+    preference :test_mode, :boolean, default: true
```
Comment (curtiscook) [inline app/models/spree/gateway/affirm.rb]: Can we flag test_mode orders as such?

File: app/models/spree/gateway/affirm.rb (hunk 4 above)
Comment (curtiscook) [reply app/models/spree/gateway/affirm.rb]: Would  be better if we could handle this w/ test_mode

File: Gemfile (hunk 1 above)
Comment (diptib10) [reply Gemfile]: done

File: app/views/spree/checkout/payment/_affirm.html.erb (hunk 2 above)
Comment (diptib10) [reply app/views/spree/checkout/payment/_affirm.html.erb]: 👍

File: app/models/spree/gateway/affirm.rb (hunk 4 above)
Comment (diptib10) [reply app/models/spree/gateway/affirm.rb]: I'll clarify this in the docs.

File: app/models/spree/gateway/affirm.rb (hunk 5 above)
Comment (diptib10) [reply app/models/spree/gateway/affirm.rb]: No

**Pull request merged**